Docs/Improved debugging.md

```diff
@@ -0,0 +1,41 @@
+# Improving debugging by bat files {#improvedebug}
+
+If you do not want to use your terminal enormous amount of times while you are debugging or testing new features it can help you
+
+You can use .bat files to run server or server & client on your own PC already with arguments and by 2 clicks of mouse
+
+#### 1. Creation of server.start .bat file
+1. Create an empty .txt file where you wish
+2. Copy the next piece of text and paste it into already created .txt file
+
+	```c++
```

Comment:
is that c++ code?

Reply:
I think, yes

Reply:
Of course no, I just found this tip in our docs, I decided to underline text and it was only way that knew
Maybe I'll find better ways about this point